src/Cli.Tests/ConfigGeneratorTests.cs

```diff
@@ -189,7 +190,9 @@ public void TestSpecialCharactersInConnectionString()
 
         // Comparing explicit strings here since parsing these into JSON would lose
         // the test scenario of verifying escaped chars are not written to the file system.
-        Assert.AreEqual(expectedRuntimeConfigJson.ToString(), actualRuntimeConfigJson.ToString());
+        string a = expectedRuntimeConfigJson.ToString();
```

Comment:
nit: I think this can be reverted to how it was earlier

Reply:
oops yes, a testing change. Thanks for spotting!